commands.js

```diff
@@ -916,25 +916,25 @@ exports.commands = {
 		if (group) {
 			buffer.push('Global auth: ' + group.charAt(0));
 		}
-		for (let i = 0; i < Rooms.global.chatRooms.length; i++) {
-			let curRoom = Rooms.global.chatRooms[i];
+		for (let id in Rooms.rooms) {
```

Comment:
Not sure if this is worth it. Maybe restrict it to active battles the user has joined?